src/main/java/com/minecolonies/coremod/event/ClientEventHandler.java

```diff
@@ -82,8 +80,7 @@ public void renderWorldLastEvent(@NotNull final RenderWorldLastEvent event)
                 final Entity entityCitizen = world.getEntityByID(citizen.getEntityId());
                 if (entityCitizen instanceof EntityCitizen)
                 {
-                    RenderUtils.renderSigns(world, event.getPartialTicks(), citizen, player,
-                      entityCitizen.getPosition());
+                    //TODO: Reimplement status rendering.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)